readme.md

```diff
@@ -667,18 +660,17 @@ Array.method('reduce', function (parameters){
 * Accessing a missing element will give you `undefined`
 * If you have an algorithm that relies on the array not being empty and not having `undefined` values, you can write a function that will prep your array to have a certain number of defined values, essentially initializing it with certain values in place
 	* An `Array.dim` function is oulined on page 63 which will allow `var myArray = Array.dim(10,0)` to make an array with 10 zeroes starting from the first position in the array(0)
-* Javascript only has **one dimensional arrays** but **_can_ have arrays of arrays**
-* Two dimensional arrays (matricess) will have to be set up by the programmer
+* Javascript only has **one dimensional arrays** but **_can_ have array of arrays**
```

Comment:
I don't think this one was a typo, but I also don't think it changes the meaning here so no problem! 👍